src/lib/converter/nodes/variable.ts

```diff
@@ -30,6 +30,7 @@ export class VariableConverter extends ConverterNodeComponent<ts.VariableDeclara
 
     /**
      * Analyze the given variable declaration node and create a suitable reflection.
+     * TODO: the type of `node` is incorrect, it should be a union of ts.PropertySignature | ts.PropertyDeclaration | ...
```

Comment:
Leaving resolving this to a later PR since changing to the actual type caused quite a few type errors.